tests/include/drivers/test_driver.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Umbrella include for all of the test driver functionality
+ */
+/*  Copyright The Mbed TLS Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *  not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#ifndef PSA_CRYPTO_TEST_DRIVER_H
+#define PSA_CRYPTO_TEST_DRIVER_H
+
+#define PSA_CRYPTO_TEST_DRIVER_LIFETIME 0x7fffff
```

Comment:
We'll have multiple test drivers, so it would be better to get into the habit of giving them some kind of name, even if it's only `TEST_DRIVER_1`.

Reply:
Are there really going to be more? Keep in mind this lifetime value is only relevant for the opaque drivers, since transparent doesn't use lifetime. It'll at some point also have to be merged into the autogenerated file, since I recall you wanted to have the location lifetime values declared in JSON?

Reply:
We'll need to test builds with multiple drivers. At least some of the test drivers will have their expected location hard-coded so that they can check that they're being called with the correct location (dispatching to the wrong driver is a plausible mode of failure that we should test).

Reply:
I agree with wanting to test with multiple drivers, but don't you think that's a task for when testing with multiple drivers actually becomes feasible? Read: for when autogen is in place and defining a hardcoded location will actually be possible? This is just a placeholder...